projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import { jobData } from '../jobData';
+import { VdkOption } from '../vdkOptions/vdk_options';
+
+export interface IDeployJobDialogProps {
+    /**
+     * Current Job name
+     */
+    jobName: string;
+    /**
+     * Current Team name
+     */
+    jobTeam: string;
+    /**
+    * Current Job path
+    */
+    jobPath: string;
+}
```

Comment:
What is the diff of backing interface of `IDeployJobDialogProps` with `ICreateJobDialogProps`?

Reply:
I will create a common one and delete these.

Reply:
These will be changed when we introduce direct fetching of jobName jobTeam from config.init just 1-2 common ones can be introduced instead of multiple